front/src/config/i18n/fr.json

```diff
@@ -1609,6 +1610,16 @@
     "invalidIcon": "L'icône est requise",
     "sceneAlreadyExist": "Une scène avec le même nom existe déjà."
   },
+  "duplicateScene": {
+    "cardTitle": "Dupliquer la scène \"{{name}}\"",
+    "nameLabel": "Nom",
+    "namePlaceholder": "Entrez un nouveau nom pour la scène dupliquée, celui-ci doit être différent du nom de scène précédent.",
```

Comment:
Title is too long for a placeholder, we can't read it: 

<img width="433" alt="Screenshot 2022-12-16 at 11 33 57" src="https://user-images.githubusercontent.com/7365207/208022918-b407914e-ba31-4e2f-8e6c-62a08502c787.png">

Why not:

> "Entrez un nouveau nom"

If you want to add an explanation, it should be added outside of the placeholder